### PR TITLE
docs(opentelemetry): fix todos

### PR DESCRIPTION
**Pull request by sumo-drosiek**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/data-source-and-configurations/index.md [hunk 1]
```diff
@@ -31,7 +33,7 @@ Some of the sources supported by Sumo Logic OoenTelemetry Collectors include:
 
 Overall, receivers and sources are key components in the Sumo Logic Otel collector that work together to ingest and process data from various sources and protocols.
 
-## OpenTelemetry Configuration
+## OpenTelemetry Configuration (ToDo: it duplicates with additional-configurations-reference)
```
Comment (sumo-drosiek) [inline docs/send-data/opentelemetry-collector/data-source-and-configurations/index.md:36]: We need to resolve that. Not sure about the approach

File: docs/send-data/opentelemetry-collector/data-source-and-configurations/index.md [hunk 2]
```diff
@@ -49,10 +51,9 @@ The OpenTelemetry configuration is composed of the following building blocks.
 
 Otel configuration can be done using a configuration file or through environment variables. The configuration file is a YAML or JSON file that defines the configuration of the Otel collector, including the receivers, processors, exporters, and extensions. The environment variables can be used to set specific configuration options at runtime.
 
-## Sumo Logic Otel Configuration structure (TODO)
-
- `sumologic.yaml`, `conf.d` directory. How does all of this work?
+## Sumo Logic Otel Configuration structure (ToDo: it duplicates with additional-configurations-reference)
```
Comment (sumo-drosiek) [inline docs/send-data/opentelemetry-collector/data-source-and-configurations/index.md:54]: We need to resolve that. Not sure about the approach

File: docs/send-data/opentelemetry-collector/data-source-and-configurations/additional-configurations-reference.md [hunk 3]
```diff
@@ -103,7 +21,7 @@ For example, a file named `conf.d/mysql.yaml` can contain the MySQL receiver alo
...
 
 * **Memory limiter processor**. It is used to prevent out-of-memory situations on the collector. It should be always first on the processor's list. For more information, refer to the [OpenTelemetry documentation](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/memorylimiterprocessor#memory-limiter-processor).
-* **Batch processor **. It accepts spans, metrics, or logs and places them into batches. Batching helps better compress the data and reduce the number of outgoing connections required to transmit the data. See [Using batch processor to batch data](#using-batch-processor-to-batch-data) for more information.
+* **Batch processor**. It accepts spans, metrics, or logs and places them into batches. Batching helps better compress the data and reduce the number of outgoing connections required to transmit the data. See [Using batch processor to batch data](#using-batch-processor-to-batch-data) for more information.
 <!-- * TODO: verify this point -> The Sumo Logic Schema processor modifies the metadata on logs, metrics, and traces sent to Sumo Logic so that the Sumo Logic apps can make full use of the ingested data.  -->
```
Comment (JainM6) [inline docs/send-data/opentelemetry-collector/data-source-and-configurations/additional-configurations-reference.md:25]: @sumo-drosiek Who can verify this? I think this is the only one left.

**Comment by sumo-drosiek** [comment]
Merging to `structure-new` as requested by @JainM6

**Pull request merged**